src/Paket.Core/PublicAPI.fs

```diff
@@ -87,6 +87,25 @@ type Dependencies(dependenciesFileName: string) =
         deps.DownloadLatestBootstrapper()
 #endif
 
+    /// Initialize paket.dependencies file in the given directory
+    static member Init(directory, sources, additional, downloadBootstrapper) =
+        let directory = DirectoryInfo(directory)
+
+        RunInLockedAccessMode(
+            directory.FullName,
+            fun () ->
+                PaketEnv.initWithContent sources additional directory
+                |> returnOrFail
+        )
+
+#if !NO_BOOTSTRAPPER
```

Comment:
I don't even know that flag?

Reply:
Added that to disable boostrapper on .net core. was not working anyway, no need to download it